### PR TITLE
Ignore auditd error in loganalyzer

### PR DESCRIPTION
**Pull request by maipbui**
### Description of PR

Summary:
Fixes # (issue)

### Type of change

- [ ] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)

### Back port request
- [ ] 202012
- [ ] 202205
- [ ] 202305
- [ ] 202311
- [x] 202405

### CODE REVIEW

File: ansible/roles/test/files/tools/loganalyzer/loganalyzer_common_ignore.txt [hunk 1]
```diff
@@ -304,3 +304,5 @@ r, ".* ERR syncd\d*#syncd.*SAI_API_BUFFER.*Unsupported buffer pool.*"
 
 # Ignore auditd error
 r, ".* ERR auditd\[\d*\]: Error receiving audit netlink packet \(No buffer space available\)"
+r, ".* ERR audisp-tacplus: tac_connect_single: connection failed with.*Interrupted system call"
+r, ".* ERR audisp-tacplus: tac_connect_single: connection failed with.*Transport endpoint is not connected"
```
Comment (maipbui) [inline ansible/roles/test/files/tools/loganalyzer/loganalyzer_common_ignore.txt:308]: These ERR are not related to recent auditd rule changes, it already exists in prod since auditd is enabled for per-command accounting feature.

**Review by yejianquan — APPROVED** [review]
LGTM

**Comment by agadia-cisco** [comment]
@yejianquan could you please cherry pick the PR to `202411` too.

**Comment by Verma-Anukul** [comment]
@maipbui 
Please help in merging his PR to 202411 branch too

**Comment by maipbui** [comment]
Cherry-pick PR to 202411: https://github.com/sonic-net/sonic-mgmt/pull/19767
